curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-dom-manipulation-by-building-a-role-playing-game/62a3cfc8328d3351b95d4f61.md

```diff
@@ -1,26 +1,25 @@
 ---
 id: 62a3cfc8328d3351b95d4f61
-title: Step 42
+title: Step 36
 challengeType: 0
-dashedName: step-42
+dashedName: step-36
```

Comment:
It would be best to open up a separate issue to address when to use backticks in this certification. That way the team can decide on the best case moving forward with all projects.

Because some tests will look for `('"`)` while others won't. So that needs to be a separate discussion because it affects more then just this project.